Packs/Maltiverse/Integrations/Maltiverse/Maltiverse.yml

```
@@ -0,0 +1,415 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Maltiverse
+  version: -1
+configuration:
+- display: API Key
+  name: api_key
+  required: false
+  type: 12
```

```suggestion
  type: 4
```
Generally, we want to keep API keys secret.